lib/rbUtil.js

```diff
@@ -199,4 +180,11 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+// Store titles as MediaWiki db keys
```

Comment:
Sure, sounds good to me. Lets add a comment about this & group the MW bits together so that we don't forget.